lib/src/component/error_boundary.dart

```diff
@@ -384,4 +427,36 @@ class ErrorBoundaryComponent<T extends ErrorBoundaryProps, S extends ErrorBounda
   /// were thrown by the exact same component within the tree.
   String _getReadableErrorInfo(/*NativeJavascriptObject*/dynamic jsErrorInfo) =>
       getProperty(jsErrorInfo, 'componentStack');
+
+  /// The logger that logs errors when a component somewhere within the React tree wrapped by
+  /// this [ErrorBoundary] instance throws within the React lifecycle.
+  Logger get logger => _logger;
+  Logger _logger;
+
+  String _getLoggerName([T propsMap]) {
+    propsMap ??= props;
+    if (propsMap.loggerName != null && propsMap.loggerName.isNotEmpty) return propsMap.loggerName;
```

Comment:
Why an `isNotEmpty` check? The empty string is a valid logger name. Plus, in general, I don't think we should support multiple "nully" values in components unless it significantly improves usability, since that's not really idiomatic Dart.